File: CS/dxSample_CustomSimpleButton/Form1.cs
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Linq;
using DevExpress.LookAndFeel;
using DevExpress.Skins;

namespace dxSample_CustomSimpleButton {
    public partial class Form1 : DevExpress.XtraEditors.XtraForm {
        public Form1() {
            InitializeComponent();
        }
        private void InitSimpleButton() {
            var skin = GetCurrentSkin();
            SetElementImage(skin, GetImage());
            LookAndFeelHelper.ForceDefaultLookAndFeelChanged();
        }
        private void SetElementImage(Skin skin, Image img) {
            if (img == null) {
                return;
            }
            var element = skin[CommonSkins.SkinButton];
            element.Image.SetImage(img, System.Drawing.Color.Empty);
        }
        private Image GetImage() {
            return global::dxSample_CustomSimpleButton.Properties.Resources.button1;
        }
        private Skin GetCurrentSkin() {
            return CommonSkins.GetSkin(UserLookAndFeel.Default.ActiveLookAndFeel);
        }
        private void Form1_Load(object sender, EventArgs e) {
            InitSimpleButton();
        }
    }
}

File: VB/dxSample_CustomSimpleButton/Form1.vb
Imports System
Imports System.Collections.Generic
Imports System.Drawing
Imports System.Linq
Imports DevExpress.LookAndFeel
Imports DevExpress.Skins

Namespace dxSample_CustomSimpleButton
    Partial Public Class Form1
        Inherits DevExpress.XtraEditors.XtraForm

        Public Sub New()
            InitializeComponent()
        End Sub
        Private Sub InitSimpleButton()
            Dim skin = GetCurrentSkin()
            SetElementImage(skin, GetImage())
            LookAndFeelHelper.ForceDefaultLookAndFeelChanged()
        End Sub
        Private Sub SetElementImage(ByVal skin As Skin, ByVal img As Image)
            If img Is Nothing Then
                Return
            End If
            Dim element = skin(CommonSkins.SkinButton)
            element.Image.SetImage(img, System.Drawing.Color.Empty)
        End Sub
        Private Function GetImage() As Image
            Return My.Resources.button1
        End Function
        Private Function GetCurrentSkin() As Skin
            Return CommonSkins.GetSkin(UserLookAndFeel.Default.ActiveLookAndFeel)
        End Function
        Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
            InitSimpleButton()
        End Sub
    End Class
End Namespace
